presto-docs/src/main/sphinx/release/release-0.221.rst

```diff
@@ -0,0 +1,54 @@
+=============
+Release 0.221
+=============
+
+General Changes
+---------------
+* Fix a null pointer exception during the stats collection phase in query planning.
+* Fix an issue where outer joins without equality predicates would sometimes run on a single node
```

Comment:
I think this is too low level for the release notes, but I'm not sure how to phrase it better.

Reply:
I could be less specific? e.g. 
Fix a performance regression for some outer joins without equality predicates when ``join_distribution_type`` is set to ``AUTOMATIC``.